cmd/livepeer/starter/starter.go

```diff
@@ -1191,11 +1194,29 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 			currencyBase = currency
 		}
 
-		if *cfg.AIModels != "" {
-			configs, err := core.ParseAIModelConfigs(*cfg.AIModels)
-			if err != nil {
-				glog.Errorf("Error parsing -aiModels: %v", err)
-				return
+		if *cfg.AIModels != "" && *cfg.AIModelsWebhookUrl != "" {
+			glog.Error("Both '-aiModels' and '-aiModelsWebhookUrl' flags are set. Please specify only one of them.")
+			return
+		} else if *cfg.AIModelsWebhookUrl != "" || *cfg.AIModels != "" {
+			var configs []core.AIModelConfig
+			var err error
+
+			if *cfg.AIModelsWebhookUrl != "" {
+				webhook, err := core.NewAIModelWebhook(*cfg.AIModelsWebhookUrl, 1*time.Minute)
+				if err != nil {
+					glog.Errorf("Error creating AI model webhook: %v", err)
+					return
+				}
+				configs = webhook.GetConfigs()
+				n.ModelsWebhook = webhook
+			}
+
+			if *cfg.AIModels != "" {
+				configs, err = core.ParseAIModelConfigs(*cfg.AIModels)
+				if err != nil {
+					glog.Errorf("Error parsing -aiModels: %v", err)
+					return
+				}
 			}
 
 			for _, config := range configs {
```

Comment:
@gioelecerati This entire loops needs to be run after if [there were changes to the config](https://github.com/livepeer/go-livepeer/pull/3209/files#diff-85b4611f2c22c73db96c87db4253da7c72df9107f7b4c545b740f88ce2fda2e5R191)